src/core.js

```diff
@@ -755,11 +755,30 @@ define([
             }
 
             if (window.analytics) {
-                window.analytics.usage(
-                    "Question Reference",
-                    "Drag and Drop",
-                    _this.data.core.currentlyEditedProperty
```

Comment:
It might be, but the property isnt part of the new spec for what @dimagi/product wanted in ga. The new spec (in the google doc linked above) cares more about the target